app/src/main/java/com/github/jorgecastillo/kotlinandroid/functional/predef.kt

```diff
@@ -0,0 +1,73 @@
+package com.github.jorgecastillo.kotlinandroid.functional
+
+import com.github.jorgecastillo.architecturecomponentssample.model.error.*
+import com.github.jorgecastillo.kotlinandroid.di.context.GetHeroesContext
+import kategory.*
+
+typealias Control<F> = MonadControl<F, GetHeroesContext, CharacterError>
```

Comment:
I (and users) are gonna need some explanation on this file if possible, since all the important stuff for this PR resides here. Could you elaborate a bit more here or inside PR description ? It would be helpful for me to review it better. Thanks!